packages/steppp-react/example/SubStep.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Step, useSteppp } from "../src";
+
+export default ({ children }) => {
```

Comment:
another great example of a place to promote good practices 💯 

```suggestion
function SubStep({ children }) {
```